esp/esp/program/modules/forms/teacherreg.py

```diff
@@ -66,6 +66,10 @@ class TeacherClassRegForm(FormWithRequiredCss):
     #     [["Lecture", "Lecture Style Class"], ["Seminar", "Seminar Style Class"]]
     style_choices = []
 
+    # To enable class styles, admins should set the Tag grade_ranges.
+    # e.g. [[7,9],[9,10],[9,12],[10,12],[11,12]] gives five grade ranges: 7-9, 9-10, 9-12, 10-12, and 11-12
+    grade_range_choices = []
```

Comment:
You can just inline this, and put the comment by the field.  (I guess you're following `style_choices` but I think that's a silly pattern.)